README.md

```diff
@@ -11,11 +11,18 @@
 
 ## Running :ocean:
 
-- Vulkan: `cargo run --features vulkan`
-- D3D12: `cargo run --features dx12`
-- Metal: `cargo run --features metal`
+- Vulkan: `cargo run --example ocean --features vulkan`
```

Comment:
Can we make it a binary instead of an example? The benefit is not having to specify it. There is a default binary, so users coudl just do `cargo run --features vulkan` like they do now.

Reply:
Fix this.

But we can't do this for ios.
```
the target `ocean_ios` is a binary and can't have any crate-types set (currently "staticlib")
```

Reply:
ah sorry!